1-exercises/H-array-methods-2/exercise2.js

```diff
@@ -7,7 +7,13 @@
   Tip: use the string method .split() and the array method .join()
 */
 
-function capitalise(str) {}
+function capitalise(str) {
+  const words = str.split(" ");
+  for (let i = 0; i < words.length; i++) {
+    words[i] = words[i].charAt(0).toUpperCase() + words[i].slice(1);
+  }
+  return words.join(" ");
+}
```

Comment:
Well done. Now you should try to rewrite it without using a loop but using `map()` instead. The code will look much better.